### ISSUE TITLE
Export result to CSV format

### ISSUE THREAD

**Issue by nguyenxuanluong-luvina** [darold/ora2pg#1022]
Hi darold, 
Thank for your very useful tools, it's save so much time!
I have some table need export to CSV file, I read in document and see a flag : --dump_as_csv but seem it's relate to report, not data export.
Are there any way to achive the csv file?

**Comment by darold** [comment]
Yes --dump_as_csv is used for assessment reports and there is no CSV output for data. You can easily use the COPY output to convert it as CSV through a scripts, the separator is tabulation.

**Comment by nguyenxuanluong-luvina** [comment]
@darold Thank you, I will try to custome the COPY output.
But it's greate if this feature available in next releases!

**Comment by RonJohn2** [comment]
@Ryo51289 the pg_bulkload SKIP option lets you bypass the those pesky lines.
